<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Animate</class>
 <widget class="QMainWindow" name="Animate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>405</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>宋体</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Image transition</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="btnImage1">
        <property name="text">
         <string>Choose Image &amp;1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="image1">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string>Image 1</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="btnImage2">
        <property name="text">
         <string>Choose Image &amp;2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="image2">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string>Image 2</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="btnAnimate">
        <property name="text">
         <string>&amp;Animate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="animation">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string>Animation</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
